lte/gateway/deploy/agw_post_install.sh

```diff
@@ -43,13 +43,20 @@ if [ "$PING_RESULT" != "$SUCCESS_MESSAGE" ]; then
     addError "eth0 is connected to the internet" "Make sure the hardware has been properly plugged in (eth0 to internet)"
 fi
 
-allServices=("control_proxy" "directoryd" "dnsd" "enodebd" "magmad" "mme" "mobilityd" "pipelined" "policydb" "redis" "sctpd" "sessiond" "state" "subscriberdb")
+allServices=("control_proxy" "directoryd" "dnsd" "enodebd" "magmad" "mme" "mobilityd" "pipelined" "policydb" "redis" "sessiond" "state" "subscriberdb")
 for service in "${allServices[@]}"; do
     if ! systemctl is-active --quiet "magma@$service"; then
         addError "$service is not running" "Please check our faq"
     fi
 done
 
+notMagmaServices=("sctpd")
```

Comment:
Sctpd is still a Magma project service, it is just packaged separately than other services, and named differently. Can we call this set magmaSctpdServices as they are part of a different package?

Reply:
I wanted a name for all the services that are not managed by magmad

Reply:
Ah, then let's do
```suggestion
nonMagmadServices=("sctpd")
```

Reply:
sounds good to me thanks for the great name :)